demisto_sdk/runners/playbook_runner.py

```
@@ -0,0 +1,112 @@
+                if playbook_results["state"] == "inprogress":
+                    time.sleep(10)
+                    elasped_time = int(time.time() - start_time)
+                else:   # the playbook has finished running
+                    break
+
+            # Ended the loop because of timeout
+            if elasped_time > self.timeout:
+                print_error(f'The command had timed out while the playbook is in progress.\n'
+                            f'To keep tracking the playbook please go to : {work_plan_link}')
+            else:
+                if playbook_results["state"] == "failed":
+                    print_color("The playbook finished running with status: FAILED", LOG_COLORS.RED)
+                else:
+                    print_color("The playbook finished running with status: COMPLETED", LOG_COLORS.GREEN)
+
+        # The command does not wait for the playbook to finish running
+        else:
+            print(f'To see results please go to : {work_plan_link}')
+
+        return 0
+
+    def create_incident_with_playbook(self, incident_name, playbook_id):
+        # type: (str, str) -> int
+        """Create an incident in Demisto with the given incident_name and the given playbook_id
+
+        Args:
+            incident_name (str): The name of the incident
+            playbook_id (str): The id of the playbook
+
+        Returns:
+            int. The new incident's ID. Returns 1 in a case of creation error.
+        """
+
+        create_incident_request = demisto_client.demisto_api.CreateIncidentRequest()
+        create_incident_request.create_investigation = True
+        create_incident_request.playbook_id = playbook_id
+        create_incident_request.name = incident_name
+
+        try:
+            response = self.demisto_client.create_incident(create_incident_request=create_incident_request)
+        except RuntimeError as err:
+            print_error(str(err))
+            return 1
```

the id could be 1, we should change this, let's raise an error instead